### ISSUE TITLE
Polyfill does not work with direct child symbol (>)

### ISSUE THREAD

**Issue by dakur** [jonathantneal/element-qsa-scope#3]
If I open your [polyfilled test pen](https://codepen.io/jonneal/pen/0fd6772bbeb9f555cee48ed2c60163e1) in MS Edge v44 (EdgeHTML v18) and change `:scope div a` to `:scope > div a`, nothing changes visibly (test still says to pass), but in devtools console, there is an error: `SCRIPT0: SCRIPT0: Expected identifier, string or number`.

![image](https://user-images.githubusercontent.com/9862581/84622674-def7ab00-aedd-11ea-9e59-b7705de90c32.png)

**Comment by dakur** [comment]
Actually, it made the same thing when I change `:scope div a` to `:scope xyz a`, so I'm not pretty sure what is going on in there. But still it does not work on Edge.

**Comment by dakur** [comment]
When I downloaded the tests from codepen and opened locally in Edge, it worked. So I narrowed my code even more and I found out the problem is probably rather in tinymce, jquery and/or its sizzle library.

As I've written some custom code into tinymce's `setup` callback, I was expecting this polyfill should work in there as well, but it did not. So maybe tinymce is overwriting there something, I don't know. Fixed it with extracting this polyfill's behaviour directly into my code (the random attribute hack).
